fidelius-service/pom.xml

```diff
@@ -120,6 +120,7 @@
 			<groupId>software.amazon.awssdk</groupId>
 			<artifactId>auth</artifactId>
 		</dependency>
+		</dependency>
```

Comment:
This seems like an extra </dependency> tab

Reply:
Woops sorry, fixed it